assets/components/payPalContributionButton/payPalContributionButton.jsx

```diff
@@ -22,6 +22,7 @@ type PropTypes = {
   isoCountry: IsoCountry,
   errorHandler: (string) => void,
   canClick?: boolean,
+  copy?: string,
```

Comment:
Would "label" be a better name for this variable?

Reply:
Possible confusion with the `<label>` element? Buttons can have labels...